engine/action_xpub.go

```diff
@@ -19,6 +19,8 @@ func (c *Client) NewXpub(ctx context.Context, xPubKey string, opts ...ModelOps)
 		xPubKey, c.DefaultModelOptions(append(opts, New())...)...,
 	)
 
+	xPub.NextExternalNum = 1 // 0 is reserved for paymail adress external Xpub
```

Comment:
For developers less familiar with this - like myself - there should also be a comment 'Why 1.?'"

Reply:
Not relevant anymore